build/cmake/DefineOptions.cmake

```diff
@@ -160,6 +162,8 @@ message(STATUS "  Build as3 library:                          ${BUILD_AS3}")
 MESSAGE_DEP(WITH_AS3 "Disabled by WITH_AS3=OFF")
 MESSAGE_DEP(HAVE_COMPC "Adobe Flex compc was not found - did you set env var FLEX_HOME?")
 message(STATUS)
+message(STATUS "  Build with OpenSSL:                         ${WITH_OPENSSL}")
```

Comment:
Including the OpenSSL version would be nice here.

Reply:
done